main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "fmt"
+
+func max(words []string) string {
+	maxLength := len(words[0])
+	maxWordId := 0
+	for i, n := range words {
+		if maxLength < len(n) {
+			maxLength = len(n)
+			maxWordId = i
+		}
+	}
+	return words[maxWordId]
+}
+
+func reverse(numbers []int64) []int64 {
+	var reversedNumbers []int64
+	for _, n := range numbers {
+		reversedNumbers = append([]int64{n}, reversedNumbers...)
```

Comment:
Very strange form. 
Please use more obvious way to append elements to the slice:

`reversedNumbers = append(reversedNumbers, ...)`

Reply:
Please fix it.